section3/exercises/arrays.js

```diff
@@ -106,11 +125,21 @@ var hasCharger = true;
 var chargerType = "car";
 
 // Write your conditional here
+//if user has 15% or more batter remaining, no other condition must be evaluated
+if (percentBatteryLeft >= 15) {
+  console.log('Your Uber has been called!');
+//if less than 15%, they must have a charger AND it must be for a car
+} else if (percentBatteryLeft < 15 && hasCharger && chargerType === 'car') {
```

Comment:
The part `percentBatteryLeft < 15` in this line isn't necessary because of your first `if` clause.